pom.xml

```diff
@@ -55,7 +55,7 @@
         <dep.drift.version>1.33</dep.drift.version>
         <!-- Changing joda version changes tzdata which must match deployed JVM tzdata
              Do not change this without also making sure it matches -->
-        <dep.joda.version>2.10.8</dep.joda.version>
+        <dep.joda.version>2.12.2</dep.joda.version>
```

Comment:
According the comment above, changing the joda version will require to make changes to the deployed JVM tzdata, any action needed from this perspective?

Reply:
I am not sure how one can check the deployed JVM runtime, as Presto has minimal requirements for this runtime.  I think this message is meant to inform Presto operators that they'll also need to update their JRE/JVM to also include the updated tzdata whenever we update the Joda version.  The way we inform operators of such changes is through release notes.  I've updated the release note comment to make this clear and explicit.